### PR TITLE
Configurable instructions language

### PR DESCRIPTION
**Pull request by doflip**
Usage example:
    L.mapbox.directions({language: 'pt'});

The default remains english language.

### CODE REVIEW

File: src/format.js [hunk 1]
```diff
@@ -24,5 +24,31 @@ module.exports = {
...
+        m = m.replace(/\bEast/g, "Este");
+        m = m.replace(/\bWest/g, "Oeste");
+        m = m.replace(/\bonto/g, "para");
+        m = m.replace(/\bon/g, "na");
+        m = m.replace(/\bHead/g, "Siga para");
+        m = m.replace(/Turn right/g, "Vire à direita");
+        m = m.replace(/Turn left/g, "Vire à esquerda");
+        m = m.replace(/\bContinue/g, "Continue");
+        m = m.replace(/You have arrived at your destination/g, "Chegou ao seu destino");
+/*
```
Comment (TheMarex) [inline src/format.js]: This debug code should be removed.

Comment (doflip) [inline src/format.js]: yes, and I still need to add a couple of instructions, but is this the way to do it?

**Comment by freenerd** [comment]
@doflip thanks for the PR

@tmcw how do you feel about this? do you think instead of search/replacing the individual values, we could already here get https://www.transifex.com/ to work? this would down the line make it easier to migrate the changes from mapbox-directions.js to the mapbox directions API (where this eventually should live). Since @doflip likely doesn't want to wait until we get our shit together on the API I'm happy to have this in mapbox-directions.js for a while ...

**Comment by tmcw** [comment]
I think we should land this in the directions API first, rather than it hanging out in the mapbox-directions.js codebase intermediately. Free to use this PR as a fork in the meantime, but we should expedite doing this the right way, on the server side.

**Comment by tmcw** [comment]
Closing this PR, but keeping the branch alive if someone wants to start from it to do client-side translations for the time being: the server-side approach is the plan.

**Pull request closed without merging**